crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -423,6 +423,7 @@ spec:
     cache_namespaces:
     - ".*"
     cache_token_namespace_duration: 10
+    token_expire_duration: 60
```

Comment:
See: https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting ... `Ensure each group of settings are alphabetically sorted in all the files where the new setting was added`

All the settings in all these files are sorted alphabetically. So you want to move this setting after the "qps" setting.

Reply:
Changed